scipy/signal/_signaltools.py

```diff
@@ -3934,8 +3965,16 @@ def vectorstrength(events, period):
         fixed.  Biol Cybern. 2013 Aug;107(4):491-94.
         :doi:`10.1007/s00422-013-0560-8`.
     '''
-    events = np.asarray(events)
-    period = np.asarray(period)
+    if isinstance(events, (int, float)) and isinstance(period, (int, float)):
```

Comment:
Is this required now https://github.com/data-apis/array-api-compat/pull/147 has merged?